torch/_inductor/ir.py

```diff
@@ -3942,20 +3941,22 @@ def get_read_writes_input(self, x):
         return dependencies.StarDep(x.get_name())
 
     def get_read_writes(self):
-        star_dep = []
+        reads: Set[dependencies.Dep] = set()
+        StarDep = dependencies.StarDep
         for input in self.inputs:
             if isinstance(input, list):
-                star_dep.extend([self.get_read_writes_input(x) for x in input])
+                reads.update({StarDep(x.get_name()) for x in input})
```

Comment:
```suggestion
                reads.update(StarDep(x.get_name()) for x in input)
```
no need for temporary set, can take an iterable/generator directly